demo/client_demo/integration_demo/progress_demo.py

```diff
@@ -127,15 +127,17 @@ def get_data_id(size: int, data_num):
 
                     pbars[key] = (pbar, procedure.progress)
 
-        if get_res["results"] is not None:
-            for key in pbars:
-                pbar, prev = pbars[key]
-                if pbar is None:
-                    continue
-                pbar.update(100 - prev)
-                pbar.clear()
-                pbar.close()
-
+        if all([r == JobStatus.Value('COMPLETED')
+                for r in get_status["statuses"]]):
+            res = qmpc.get_computation_result(job_uuid)
+            if res["results"] is not None:
+                for key in pbars:
+                    pbar, prev = pbars[key]
+                    if pbar is None:
+                        continue
+                    pbar.update(100 - prev)
+                    pbar.clear()
+                    pbar.close()
```

Comment:
I thought it was unnecessary because the result can be obtained as long as the status is COMPLETED.
If this situation could happen, please ignore it!
```suggestion

```